great_expectations/data_context/data_context/bridge_file_data_context.py

```diff
@@ -0,0 +1,149 @@
+import logging
+from typing import Mapping, Optional, Union
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.core import ExpectationSuite
+from great_expectations.data_context.data_context.abstract_data_context import (
+    AbstractDataContext,
+)
+from great_expectations.data_context.data_context_variables import (
+    DataContextVariableSchema,
+    FileDataContextVariables,
+)
+from great_expectations.data_context.types.base import (
+    DataContextConfig,
+    datasourceConfigSchema,
+)
+from great_expectations.data_context.types.resource_identifiers import (
+    ExpectationSuiteIdentifier,
+)
+from great_expectations.datasource.datasource_serializer import (
+    YAMLReadyDictDatasourceConfigSerializer,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class BridgeFileDataContext(AbstractDataContext):
```

Comment:
I thought of this after we discussed, but maybe we make this private so that it doesn't get used
```suggestion
class _BridgeFileDataContext(AbstractDataContext):
```